### PR TITLE
MF-356 MF-358

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs [hunk 1]
```diff
@@ -20,26 +23,28 @@ public class WhenIGetTrustedEmployers
         public const uint ExpectedUkPrn = 12345;
 
         private GetTrustedEmployersQueryHandler _handler;
-        private Mock<IProviderPermissionsService> _providerPermissionsService;
+        private Mock<IProviderService> _providerService;
         private GetTrustedEmployersQuery _query;
-        private IList<Employer> _expectedEmployers;
+        private IList<AccountLegalEntity> _expectedEmployers;
```
Comment (scottywakefield) [inline src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs]: worth renaming the var to `_expectedAccountLegalEntities`, as well?

File: src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs [hunk 2]
```diff
@@ -51,12 +56,12 @@ public void Arrange()
                 UkPrn = ExpectedUkPrn
             };
 
-            _providerPermissionsService = new Mock<IProviderPermissionsService>();
+            _providerService = new Mock<IProviderService>();
             _validator = new Mock<IValidator<GetTrustedEmployersQuery>>();
 
-            _handler = new GetTrustedEmployersQueryHandler(_providerPermissionsService.Object, _validator.Object);
+            _handler = new GetTrustedEmployersQueryHandler(_providerService.Object, _validator.Object);
```
Comment (scottywakefield) [inline src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs]: GetTrustedEmployersQueryHandler => GetTrustedAccountLegalEntitiesQueryHandler
plus all other files/classes - request, response, test files, validator..

File: src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs (hunk 1 above)
Comment (dashton82) [reply src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs]: Updated

File: src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs (hunk 2 above)
Comment (dashton82) [reply src/SFA.DAS.Reservations.Application.UnitTests/Employers/Queries/WhenIGetTrustedEmployers.cs]: I think I would be tempted to keep the name of this as it does describe the action that is occurring.

**Pull request merged**